open_parser/base.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
nit: https://google.github.io/styleguide/pyguide.html is some good resource to study python style guide to improve the readability of your code.

Let's add typehint and docstring for your code to help improve open source readability for both class and methods.